src/components/tipRecords/TipRecord.vue

```diff
@@ -206,45 +210,73 @@ export default {
       return isPinned !== -1;
     },
   },
+  mounted() {
+    if (!this.$route.name === 'tip') {
```

Comment:
it shouldn't work this way, `this.$route.name` will be negated firstly
```suggestion
    if (this.$route.name !== 'tip') {
```

Reply:
actually, I'm not sure

Reply:
```js
> !('a' === 'b')
true
> !'a' === 'b'
false
```

Reply:
You're right.